include/nbl/core/hash/blake.h

```diff
@@ -21,6 +21,8 @@ struct blake3_hash_t final
 	uint8_t data[BLAKE3_OUT_LEN];
 };
 
+const blake3_hash_t INVALID_HASH = {};
+
```

Comment:
either call it `EmptyHash` or move to `IAsset`

Reply:
moved to `IPreHashed`

Reply:
you only moved to a header, but you didn't move it to become a static const variable!